genesis/genesis.go

```diff
@@ -36,18 +40,17 @@ import (
 
 // FromConfig returns:
 // 1) The byte representation of the genesis state of the platform chain
-//    (ie the genesis state of the network)
+//    (ie the genesis state of the network), formatted using the hex encoder
```

Comment:
I don't think the added comment ", formatted using the hex encoder" makes sense here, as the result is a byte array, not a string.